exchange/bitswap/bitswap_test.go

```diff
@@ -108,7 +108,7 @@ func TestLargeSwarm(t *testing.T) {
 	if detectrace.WithRace() {
 		// when running with the race detector, 500 instances launches
 		// well over 8k goroutines. This hits a race detector limit.
-		numInstances = 100
+		numInstances = 75
```

Comment:
Max go routines with race detector: 8196.
Bitswap workers per peer: n^2.

75 means 75^2 = 5625

That gives us some healthy head-room for other random goroutines.